src/Unleash/DefaultUnleash.cs

```diff
@@ -81,9 +49,6 @@ public DefaultUnleash(UnleashSettings settings, bool overrideDefaultStrategies,
             }
         }
 
-        /// <inheritdoc />
-        public ICollection<FeatureToggle> FeatureToggles => services.ToggleCollection.Instance.Features;
```

Comment:
Removing this is a breaking change, so should perhaps be pointed out in a major release. It's not in the documentation though, so nothing to update there

Reply:
Yep, noted this in the PR description, and we're planning on a major release. Something that me and @sighphyre discussed is that we'll probably need to prepare a migration guide or similar, but out of scope for this PR.